flink/v1.12/flink/src/main/java/com/netease/arctic/flink/read/source/log/kafka/LogKafkaPartitionSplitReader.java

```diff
@@ -16,17 +16,16 @@
  * limitations under the License.
  */
 
-package com.netease.arctic.flink.read.source.log;
+package com.netease.arctic.flink.read.source.log.kafka;
```

Comment:
IMO, moving the Kafka logstore implementations to another package path would cause cross-version compatibility issues. WDYT?

Reply:
Flip 27 refactor only released in version 0.4.1, so I think there is no cross-version.